experiment/resources/dispatcher-startup-script-template.sh

```diff
@@ -13,6 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Hack because container-optmized-os doesn't support writing to /home/root
```

Comment:
Maybe add the thing about you are storing docker credentials to make it clear what is the intent.